lib/apiservers/portlayer/swagger.json

```diff
@@ -3298,6 +3298,12 @@
 						"type": "string"
 					}
 				},
+        			"cmd": {
+					"type": "array",
+					"items": {
+						"type": "string"
+					}
+				},
```

Comment:
There shouldn't be a cmd in the ContainerConfig struct. Sessions have their own world (see ProcessConfig)